.github/workflows/test.yml

```diff
@@ -48,12 +46,7 @@ jobs:
 
       - name: Install Dependencies
         run: |
-          if go help mod > /dev/null 2>&1 ; then
-            go mod download
-          else
-            curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-            dep ensure
-          fi
+          go mod download
         env:
           GO111MODULE: "on"
```

Comment:
Don't need this env setting for 1.16+ which is what this build targets

Reply:
Good call out, thanks! I've removed it.